modules/__init__.py

```diff
@@ -24,9 +24,10 @@ def process_query(input):
             'Authorization': 'Bearer %s' % WIT_AI_ACCESS_TOKEN
         })
         data = r.json()
-        intent = data['outcomes'][0]['intent']
+        intent = data['outcomes'][0]['intent'] if 'intent' not in data['outcomes'][0]['entities'] else data['outcomes'][0]['entities']['intent'][0]['value']
```

Comment:
If we could follow Could we follow PEP8 please, https://www.python.org/dev/peps/pep-0008/#id19 then that would be greate